homework/style.css

```diff
@@ -1,3 +1,93 @@
 .alert-error {
```

Comment:
You CSS still needs work to make your app responsive:

![image](https://user-images.githubusercontent.com/2788771/62781411-407a0c80-bab8-11e9-867a-9dd0989cd3ef.png)